### DISCUSSION TITLE
Remove "Media Tag" from Create New Document menu?

### DISCUSSION THREAD

**Discussion by joaodotwork** [sanity-io/sanity-plugin-media#43] [Q&A] [answered]
Hi! What a wonderful plugin, thank you so much for making this Open-source and awesome :)

Quick q: I tend to edit my available types in the "Create New Document" menu and realize creating Media Tags is useful to me in the plugin window context but not outside of it.

How would you suggest me to edit it out of that list?

<img width="895" alt="CleanShot 2021-06-07 at 08 43 30@2x" src="https://user-images.githubusercontent.com/50487816/120971970-89ab4e00-c76d-11eb-81fb-d2b357a8268f.png">

Thanks + much appreciated,
João

**Comment by robinpyon** [comment, marked as answer]
Hey @joaodotwork – you can configure the new document menu using Sanity's structure builder. [Link to documentation](https://www.sanity.io/docs/initial-value-templates#56b4073ca73a)

Simply add the following to the `parts` array in your `sanity.json`:

```javascript
{
  "parts": [
    {
      "name": "part:@sanity/base/new-document-structure",
      "path": "./newDocumentStructure.js"
    }
  ]
}
```

And in `./newDocumentStructure.js`

```javascript
import S from '@sanity/base/structure-builder'

export default [
  // Bring in default values
  ...S.defaultInitialValueTemplateItems()
    // Filter out document types of `media.tag`
    .filter(template => {
      if (template.spec.id === 'media.tag') {
        return false
      }
      return true
    })
]
```

Hope that helps!

**Reply by joaodotwork** [reply]
Thank you so much Robin! Super useful :)
